### DISCUSSION TITLE
MIB2 Stuck on boot screen

### DISCUSSION THREAD

**Discussion by NeNicYs7** [Mr-MIBonk/M.I.B._More-Incredible-Bash#109] [General]
what happened
stuck on MIB2 VW boot screen

it was restarted and I can't open it anymore

remains off

please help...

**Comment by harman-f** [comment]
I misplaced my crystal ball 😉 

please let me know what you did before. 
if you used M.I.B please share log file.

**Comment by NeNicYs7** [comment]
So I installed mib2-toolbox, While checking what settings are available, I must have done something wrong because Discover Pro MIB2 is now stuck on the home screen.

The big problem is that I can't access any of the device menus, I can't go back to the green developer menu to undo what I've done, and I can't go into the service menu to flash the firmware and overwrite back to the default from works.

**Comment by harman-f** [comment]
Log file is on SD Card in folder /Backup/ 
Please share that file here.

**Comment by NeNicYs7** [comment]
but i don't have backup files

**Comment by harman-f** [comment]
Log file is automatically generated and saved on SD card while using M.I.B. 

Basic Backup is generated during installation of M.I.B on unit. 

Just check your SD card!

**Comment by NeNicYs7** [comment]
press the restart button, I don't even see anything on the black screen

how to do it, you have to repair the MIB2 unit ?

**Comment by harman-f** [comment]
> installed mib2-toolbox

Ah, you installed mib2-toolbox. In that case you are wrong here.

I understand, that your unit is not working. But without any usable information about what you did, there can not be any help.

Check out wibwiki.one for instructions on how to use telnet connection and - on a broken unit - better look for UART.
